### ISSUE TITLE
Layout - Remove Not Working

### ISSUE THREAD

**Issue by otisito** [perkbrands/app.perkstage.co#6]
Layout - Removing a Layout (Remove button in right pane) does not work. It should remove the Layout, instead it reloads the same page. Signed in as Account Administrator. [Found 150202]

**Comment by brianropper** [comment]
Can you clarify some terminology for me?  What do you mean by layout?  For this issue, I think the problem is that when you log in to http://app.perkstage.co -> click on audience(little person icon) -> Members -> Settings of a Member.  Then in the right panel at the bottom, there is a button called Remove.  When click that and accept the pop up warning, the page redirects to http://app.perkstage.co/people_user_controller/people_administrator.  If you now click on Members again, you will see the member you tried to delete still there.  This issue is similar for Audiences and Me tabs.  Is this what this issue is referring to?

**Comment by otisito** [comment]
Hi Brian, this is actually in a different place than you've described.
From any account, hover over the Account icon (star) and select "Designs". Next, select "Layouts".

http://app.perkstage.co/layout_controller/

Once there, try selecting "Settings" for any Layout. On the right you will see a "Remove" button. From previous testing, this button was not removing (deleting) the selected Layout from the application.

Image included for reference. Hope that helps. Let me know. Thanks!

![perkcms-reference-removelayout-150723](https://cloud.githubusercontent.com/assets/11634598/8850573/d520bce0-310e-11e5-8f93-c3e85d2464d8.png)

**Comment by brianropper** [comment]
That clears things up, thanks.

**Comment by otisito** [comment]
Yes, that should have been part of the build out but apparently got missed.

I'd like for it to be consistent with other areas of the app, but I would need to look around for where it is implemented as intended.

Feel free to plug in a simple confirmation for now and I'll add it to my tracking to make consistent later.

Thanks,
Jason
(from mobile)

**Comment by dijatek** [comment]
Jason,  Brian and I actually have time to look over the bugs and have been going over this one for a while today.  It looks like this issue is more of a functionality that wasn't developed opposed to a bug in developed software.  We cannot find the delete function/code anywhere.  Question is: With your new initiatives, do you want us to build this functionality in for you or is another team already working on it.  I don't want to 1) duplicate effort or 2) make light of the work required for a fix (this one is quite a bit).  Brian will continue working on it unless we hear from you otherwise.  We should be able to put at least six hours a week towards the bugs from now on.  Dan
